docs/intro/tutorial.rst

```diff
@@ -55,318 +55,333 @@ This will create a ``tutorial`` directory with the following contents::
-  able to select things like: *the link that contains the text 'Next Page'*.
-  Because of this, we encourage you to learn about XPath even if you
-  already know how to construct CSS selectors.
+You will see something like::
 
-For working with CSS and XPath expressions, Scrapy provides the
-:class:`~scrapy.selector.Selector` class and convenient shortcuts to avoid
-instantiating selectors yourself every time you need to select something from a
-response.
+    [ ... Scrapy log here ... ]
+    2016-09-19 12:09:27 [scrapy] DEBUG: Crawled (200) <GET http://quotes.toscrape.com/page/1/> (referer: None)
+    [s] Available Scrapy objects:
+    [s]   crawler    <scrapy.crawler.Crawler object at 0x7fa91d888c90>
+    [s]   item       {}
+    [s]   request    <GET http://quotes.toscrape.com/page/1/>
+    [s]   response   <200 http://quotes.toscrape.com/page/1/>
+    [s]   settings   <scrapy.settings.Settings object at 0x7fa91d888c10>
+    [s]   spider     <DefaultSpider 'default' at 0x7fa91c8af990>
+    [s] Useful shortcuts:
+    [s]   shelp()           Shell help (print this help)
+    [s]   fetch(req_or_url) Fetch request (or URL) and update local objects
+    [s]   view(response)    View response in a browser
+    >>> 
 
-You can see selectors as objects that represent nodes in the document
-structure. So, the first instantiated selectors are associated with the root
-node, or the entire document.
+Using the shell, you can try selecting elements using `CSS`_ with the response
+object::
 
-Selectors have four basic methods (click on the method to see the complete API
-documentation):
+    >>> response.css('title')
+    [<Selector xpath=u'descendant-or-self::title' data=u'<title>Quotes to Scrape</title>'>]
 
-* :meth:`~scrapy.selector.Selector.xpath`: returns a list of selectors, each of
-  which represents the nodes selected by the xpath expression given as
-  argument.
+The result of running ``response.css('title')`` is a list-like object called
+:class:`~scrapy.selector.SelectorList`, which represents a list of
+:class:`~scrapy.selector.Selector` objects that wrap around XML/HTML elements
+and allow you to run further queries to fine-grain the selection or extract the
+data.
 
-* :meth:`~scrapy.selector.Selector.css`: returns a list of selectors, each of
-  which represents the nodes selected by the CSS expression given as argument.
+To extract the text from the title above, you can do::
 
-* :meth:`~scrapy.selector.Selector.extract`: returns a unicode string with the
-  selected data.
+    >>> response.css('title::text').extract()
+    [u'Quotes to Scrape']
 
-* :meth:`~scrapy.selector.Selector.re`: returns a list of unicode strings
-  extracted by applying the regular expression given as argument.
+There are two things to note here: one is that we've added ``::text`` to the
+CSS query, to mean that we want to select the text from inside the title element.
+If we don't specify ``::text``, we'd get the HTML tags::
```

Comment:
If I'd be reading this I'd thought that "::text" is a way to get text from a css selector, but e.g. for `"<p>hello <b>world</b></p>"` selector `p::text` will only return `['hello ']`.